src/wallet/wallet.ts

```diff
@@ -628,8 +629,8 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
    * @inner
    */
   async getUtxoFromId(txId: string, index: number): Promise<Utxo | null> {
-    const data = await walletApi.getUtxos(this, { txId, index });
-    const utxos = data.utxos;
+    const data = await walletApi.getTxOutputs(this, { txId, index });
```

Comment:
I think you should continue using the utxo API. For example, in this case here we don't want any return if it's spent, so the tx outputs API should never be used here, only the utxo.

We could force just the parameter `skipSpent: true` but for what I've seen in the wallet service implementation, if we send `txId` and `index`, you ignore the other parameters.

I think in this line here we should return only utxos with one of the solutions above, not sure which one is the best.

Reply:
As we've discussed on slack, we will use the `tx_output` API, but will always send the `skipSpent: true`. The tx_output API will only return the tx_output if it is an UTXO (spentBy is null). This has been done on the wallet-service API PR

a5300f801e24c0149aea4630e34ae5f9c8345b04